src/App.tsx

```diff
@@ -16,8 +16,12 @@ import cytoscapeSvg from "cytoscape-svg";
 import { Github, Twitter } from "./svgs";
 import useLocalStorage from "react-use-localstorage";
 
-cytoscape.use(dagre);
-cytoscape.use(cytoscapeSvg);
+
+if (!cytoscape.prototype.hasInitialised) {
+  cytoscape.use(dagre);
+  cytoscape.use(cytoscapeSvg);
+  cytoscape.prototype.hasInitialised = true;
+}
```

Comment:
Is this a common pattern in cytoscape.js apps?

Reply:
Not sure, I've never used it before. It did stop these errors from happening though when hot reloading locally

<img width="1077" alt="image" src="https://user-images.githubusercontent.com/2988301/109550333-0c65a280-7ac7-11eb-9c27-bff02ca155ae.png">

I can remove this change if you prefer

Reply:
No, thank you! That error was driving me crazy